### PR TITLE
Make fixes for workout and change sendNotfication argumetns

### PR DESCRIPTION
**Pull request by v4m3rrr**

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -189,8 +191,31 @@ const deleteUser = async (req, res) => {
...
-                await workout.deleteOne();
+                workout.user = null;
+
+                if(workout.company.length !==0 && !workout.endTime)
+                {      
+                    
+                    workout.user = workout.company[0];
+                    const newUser = await User.findById(workout.user);
+
+                    if(!newUser.workouts)
```
Comment (VEXI19) [inline controllers/user.js:204]: There is no need to add workout to the user, he already is a part of the workout so the workout id is in his workouts array

File: controllers/user.js [hunk 2]
```diff
@@ -189,8 +191,31 @@ const deleteUser = async (req, res) => {
 
         for (let i = 0; i < user.workouts.length; i++) {
             const workout = await Workout.findById(user.workouts[i]);
+
             if (workout.user.toString() === userID) {
```
Comment (VEXI19) [inline controllers/user.js:195]: this are not all the possiblities that can happen to a workout after user is deleted.
- user is the owner
    - company array is empty - delete workout
    - company array is not empty
        - workout has not happend yet - take first person from company and make them the owner
        - workout has happend - delete user from the owner, leave company as is
- user is not the owner
    - there is no owner and user is the only company - delete workout
    - there is owner - delete user id from company / access

Comment (v4m3rrr) [reply controllers/user.js:195]: You said that we store all workouts, so I don't get it.

**Comment by v4m3rrr** [comment]
I hope this is what you meant.

File: controllers/user.js (hunk 2 above)
Comment (VEXI19) [reply controllers/user.js:196]: we do, if there is a user that could access those workouts.

**Review by VEXI19 — APPROVED** [review]

File: controllers/user.js (hunk 1 above)
Comment (VEXI19) [reply controllers/user.js]: i just realized that it was implemented already, you just created your branch before the changes were made

**Pull request merged**